orbs/pulumi.yml

```diff
@@ -105,6 +141,46 @@ examples:
                 version: "16.0.2"
                 access-token: ${OTHER_ACCESS_TOKEN}
 
+  stack_init_example:
```

Comment:
Could you delete these two examples?

The `stack init` and `stack rm` examples aren't useful in and of themselves. To be practically of value within a CI/CD environment, we need to add a couple more commands. (`pulumi stack destroy` and `pulumi stack output`.)

I'll file a new issue, in case you are interested in addressing those. Since once we have those, we can complete the "end-to-end" scenario where you can use `stack init` and `rm`.